File: VB/Task26/NoteForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace Task26
	Partial Public Class NoteForm
		Inherits DevExpress.XtraEditors.XtraForm
		Public Sub New(ByVal N As Note)
			InitializeComponent()
			TxName.DataBindings.Add("EditValue", N, "Name")
			MMText.DataBindings.Add("EditValue", N, "Text")
			DtEdDate.DataBindings.Add("DateTime", N, "Date")
		End Sub
	End Class
End Namespace

File: CS/Task26/NoteForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace Task26
{
    public partial class NoteForm : DevExpress.XtraEditors.XtraForm
    {
        public NoteForm(Note N)
        {
            InitializeComponent();
            TxName.DataBindings.Add("EditValue", N, "Name");
            MMText.DataBindings.Add("EditValue", N, "Text");
            DtEdDate.DataBindings.Add("DateTime", N, "Date");
        }
    }
}